asgs.ttl

```diff
@@ -101,421 +105,429 @@ asgs:addToMbPathFrom rdf:type owl:ObjectProperty ;
 asgs:suaToSa2PathFrom rdf:type owl:ObjectProperty ;
                       rdfs:subPropertyOf asgs:from ;
                       rdfs:isDefinedBy <http://linked.data.gov.au/def/asgs> ;
                       rdfs:label "suaToSa2Path from" ;
-                      rdfs:seeAlso <http://linked.data.gov.au/def/asgs#SuaToSa2Path> .
+                      rdfs:seeAlso asgs:SuaToSa2Path .
 
 
 ###  http://linked.data.gov.au/def/asgs#suaToSa2PathTo
 asgs:suaToSa2PathTo rdf:type owl:ObjectProperty ;
                     rdfs:subPropertyOf asgs:to ;
                     rdfs:isDefinedBy <http://linked.data.gov.au/def/asgs> ;
                     rdfs:label "suaToSa2Path to" ;
-                    rdfs:seeAlso <http://linked.data.gov.au/def/asgs#SuaToSa2Path> .
+                    rdfs:seeAlso asgs:SuaToSa2Path .
 
 
 ###  http://linked.data.gov.au/def/asgs#to
 asgs:to rdf:type owl:ObjectProperty ;
         rdfs:isDefinedBy <http://linked.data.gov.au/def/asgs> ;
         rdfs:label "to" ;
-        rdfs:seeAlso <http://linked.data.gov.au/def/asgs#GeospatialPath> .
+        rdfs:seeAlso asgs:GeospatialPath .
 
 
 ###  http://linked.data.gov.au/def/asgs#trToSa2PathFrom
 asgs:trToSa2PathFrom rdf:type owl:ObjectProperty ;
                      rdfs:subPropertyOf asgs:from ;
                      rdfs:isDefinedBy <http://linked.data.gov.au/def/asgs> ;
                      rdfs:label "trToSa2Path from" ;
-                     rdfs:seeAlso <http://linked.data.gov.au/def/asgs#TrToSa2Path> .
+                     rdfs:seeAlso asgs:TrToSa2Path .
 
 
 ###  http://linked.data.gov.au/def/asgs#trToSa2PathTo
 asgs:trToSa2PathTo rdf:type owl:ObjectProperty ;
                    rdfs:subPropertyOf asgs:to ;
                    rdfs:isDefinedBy <http://linked.data.gov.au/def/asgs> ;
                    rdfs:label "trToSa2Path to" ;
-                   rdfs:seeAlso <http://linked.data.gov.au/def/asgs#TrToSa2Path> .
+                   rdfs:seeAlso asgs:TrToSa2Path .
 
 
 ###  http://linked.data.gov.au/def/asgs#uclToSa1PathFrom
 asgs:uclToSa1PathFrom rdf:type owl:ObjectProperty ;
                       rdfs:subPropertyOf asgs:from ;
                       rdfs:isDefinedBy <http://linked.data.gov.au/def/asgs> ;
                       rdfs:label "uclToSa1Path from" ;
-                      rdfs:seeAlso <http://linked.data.gov.au/def/asgs#UclToSa1Path> .
+                      rdfs:seeAlso asgs:UclToSa1Path .
 
 
 ###  http://linked.data.gov.au/def/asgs#uclToSa1PathTo
 asgs:uclToSa1PathTo rdf:type owl:ObjectProperty ;
                     rdfs:subPropertyOf asgs:to ;
                     rdfs:isDefinedBy <http://linked.data.gov.au/def/asgs> ;
                     rdfs:label "uclToSa1Path to" ;
-                    rdfs:seeAlso <http://linked.data.gov.au/def/asgs#UclToSa1Path> .
+                    rdfs:seeAlso asgs:UclToSa1Path .
+
+
+###  http://linked.data.gov.au/def/asgs#within
+asgs:within rdf:type owl:ObjectProperty ;
+            rdfs:subPropertyOf geo:sfWithin ;
```

Comment:
This is one of the main updates which does the alignment to GeoSPARQL?

Reply:
Yes - this PR just adds `asgs:within` and also ...